indico/web/client/js/jquery/widgets/jinja/permissions_widget.js

```diff
@@ -64,7 +64,7 @@ import Palette from 'indico/utils/palette';
       return [$code, $text];
     },
     _renderOtherLabel(principal, type) {
-      let iconClass, extraText;
+      let iconClass, extraText, $code;
```

Comment:
I don't think any of the changes in `_renderOtherLabel` are needed - you never call this for a `CategoryRole` type since `_renderLabel` calls `_renderRoleLabel` for `EventRole` and `CategoryRole`

Reply:
Added two functions namely : `_renderEventRoleLabel` and `_renderCategoryRoleLabel`